app/schemas/bean.pwr.imskamieskiego.data.LocalDB/1.json

```diff
@@ -37,6 +37,12 @@
             "columnName": "locationID",
             "affinity": "INTEGER",
             "notNull": true
+          },
+          {
+            "fieldPath": "hardToReach",
```

Comment:
Troszkę ta nazwa mi się kojarzy bardziej z trudnością osiągnięcia danego punktu, niż z jego penalizacją. Chodzi o to, że ten punkt nie jest trudny do osiągnięcia tylko on sam w sobie jest punktem trudnym. Trochę to mnie zmyliło, bo myślałem, że cała ścieżka dostanie oznaczenie hardToReach. Penalized?